library/xwiki

```diff
@@ -16,10 +16,10 @@ Directory: 11/postgres-tomcat
 
 Tags: 12, 12.1, 12-mysql-tomcat, 12.1-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
```

Comment:
```suggestion
Tags: 12, 12.2, 12-mysql-tomcat, 12.2-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
```

As well as below on the postgres image.

Reply:
Thanks a lot @yosifkit . I'm so glad you caught this one. It would have broken our existing 12.1 images.... @gdelhumeau I'm letting you fix it, thanks.

Reply:
@gdelhumeau I'm letting you fix it, thanks

Reply:
@yosifkit Thanks for the catch... It's fixed now, and sorry for the mistake.